### PR TITLE
Update sidebar in ADK

### PR DESCRIPTION
**Pull request by jerrycowell96**

### CODE REVIEW

File: website/sidebars.js [hunk 1]
```diff
@@ -47,18 +47,49 @@ const sidebars = {
...
+        "items": [
+          "brain/brain",
+        ],
+        "label": "Brain v1",
+        "type": "category"
+
+      },
+      {
+        "items": [
+          "intelligence-kit/v2-intro",
```
Comment (Hackerman342) [inline website/sidebars.js:62]: Missing this file?

**Review by Hackerman342 — APPROVED** [review]
Just make sure all files are there and sidebar looks good before merging.

File: website/docs/intelligence-kit/additional-hardware/additional-hardware.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (Hackerman342) [inline website/docs/intelligence-kit/additional-hardware/additional-hardware.md:1]: rm unused file?

**Review by Hackerman342 — COMMENTED** [review]
Please resolve the conflicts with main before requesting review

File: website/docusaurus.config.js [hunk 3]
```diff
@@ -99,26 +99,39 @@ async function createConfig() {
...
-	    docId: 'contribute/contribute-website',
-	    position: 'left',
-	  },
+
+    	to: 'https://farm-ng.com/pages/about-farm-ng',
+    	label: 'About Us',
+    	position: 'left',
+},
+
+	//   {
```
[the remark this answers is not in the document]
Comment (jerrycowell96) [reply website/docusaurus.config.js:108]: I agree, that it should be added to SEP as well but the Contribute page is also accessible in the Footer.

File: website/docs/intelligence-kit/overview-intel.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+![DSCF8544](https://github.com/farm-ng/amiga-dev-kit/assets/64480560/3cfbeb81-5feb-4dcf-8fbd-2fea69f1ca98)
+
+[Buy it on Farm-ng Shop](https://farm-ng.com/products/amiga-brain-kit)
+
+The Amiga Intelligence Kit is a comprehensive system designed for developers and researchers seeking to add to their Amiga's capabilities for exploration, data collection and more.
+
+## What's Included
+
+### Brain
+
```
Comment (Hackerman342) [inline website/docs/intelligence-kit/overview-intel.md:15]: This file is not going to pass pre-commit formatting checks once you merge main into your branch

Comment (jerrycowell96) [reply website/docs/intelligence-kit/overview-intel.md:15]: Oh because of the line maximum. I will make those edits.

File: website/sidebars.js [hunk 5]
```diff
@@ -47,18 +49,47 @@ const sidebars = {
...
     },
-      "mcu_kit/microcontroller-kit",
-      "brain/brain",
-      "pendant/pendant",
+    {
+      "items": [
+        "pendant/pendant"
+
+      ],
+      "label": "Pendant",
```
Comment (Hackerman342) [inline website/sidebars.js:81]: Pendant should not be in the `Development Tools` category. Should be in `Amiga Rover` IMO

File: website/docs/pendant/pendant.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
 id: pendant
-title: Pendant Overview
+title: Overview
```
Comment (Hackerman342) [inline website/docs/pendant/pendant.md:3]: Undo this change and keep it how it was

File: website/docs/intelligence-kit/overview-intel.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+This device allows you to connect the Cameras, Brain and
+RTK-GPS module to the Amiga.
+It includes 6 ports to allow connection of 4 smart cameras.
+
+### Additional Hardware
+
+1. Debug Cable
+2. Mast Hardware
+3. Productive Case
```
Comment (Hackerman342) [inline website/docs/intelligence-kit/overview-intel.md:51]: productive?
```suggestion
3. Protective Case
```

File: website/docs/intelligence-kit/brain/brain-v2.md [hunk 8]
```diff
@@ -0,0 +1,194 @@
+---
+id: brain-v2
+title: Brain v2.0
+---
+# Overview
```
Comment (Hackerman342) [inline website/docs/intelligence-kit/brain/brain-v2.md:5]: Don't just start pages with `Overview`. The top header should reflect what the page is for if it were viewed without the sidebar


```suggestion
# Brain v2.0
```

Comment (Hackerman342) [reply website/docs/intelligence-kit/brain/brain-v2.md:5]: If you've done this anywhere else, please change it

**Review by Hackerman342 — APPROVED** [review]
Requesting a few changes, then go ahead and merge
